configurations-conception-communes/boutons.html

```diff
@@ -17,14 +17,14 @@
 <link rel="stylesheet" href="../css/custom.css">
 <link rel="stylesheet" href="../css/split-h1.css">
 
-<!--<![endif]-->
+<!--<![endif]--> 
```

Comment:
Not sure what editor you're using, but EditorConfig should be removing all these on save https://github.com/canada-ca/systeme-conception/blob/master/.editorconfig#L8